### DISCUSSION TITLE
404, "message": "Resource not found"

### DISCUSSION THREAD

**Discussion by NASTZV15** [Chimildic/goofy#262] [Помощь с ошибкой]
Я выполняла шаги по инструкции https://chimildic.github.io/goofy/#/first-playlist, но при запуске createFirstPlaylist выдает ошибки:
function createFirstPlaylist() {
 // 1 - собираем
 let mixTracks = Source.getTracks([
     { name: 'Микс дня 2', id: '37i9dQZF1E3a9y3Tg5B7fp' },
     //{ name: 'Микс дня 3', id: '37i9dQZF1E39SmwIOG1PrK' },
 ]);
 let recomTracks = Source.craftTracks(mixTracks);
 let savedTracks = Source.getSavedTracks();   

 // 2 - обрабатываем
 Combiner.push(mixTracks, recomTracks);   
 Filter.removeTracks(mixTracks, savedTracks);
 Selector.keepRandom(mixTracks, 20);
 
 // 3 - создаем плейлист
 Playlist.saveWithReplace({
     // id: 'вашеId',
     name: 'Первый плейлист',
     tracks: mixTracks,
     randomCover: 'update',
 });
}
![image](https://github.com/user-attachments/assets/85507c38-b095-426f-9ab9-590bccacd0c7)
Что не так? Пыталась поискать инфу на форуме, но не нашла решения

**Comment by Chimildic** [comment]
Spotify ужесточил политику работы API. Подробнее [здесь](https://t.me/forum_goofy/565/2505). Решения пока нет.
